api/lib/application/target-profiles/target-profile-controller.js

```diff
@@ -74,8 +74,29 @@ const createBadge = async function (request, h) {
   return h.response(badgeSerializer.serialize(createdBadge)).created();
 };
 
-const copyTargetProfile = async (request) =>
-  usecases.copyTargetProfile({ targetProfileId: request.params.targetProfileId });
+const copyTargetProfile = async (request) => {
+  let copiedTargetProfileId;
+  const targetProfileIdToCopy = request.params.targetProfileId;
+  await DomainTransaction.execute(async (domainTransaction) => {
+    copiedTargetProfileId = await usecases.copyTargetProfile({
+      targetProfileId: request.params.targetProfileId,
```

Comment:
```suggestion
      targetProfileId: targetProfileIdToCopy,
```

Soit ça, soit supprimer la constante et utiliser directement `request.params.targetProfileId`.